src/components/molecules/timeline-events/order-edit/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import Avatar from "../../../atoms/avatar"
+
+type ByLineProps = {
+  user: { first_name: string; last_name: string; email: string }
+}
+export const ByLine = ({ user }) => (
+  <div className="flex inter-small-regular items-center text-grey-50">
+    By
+    <span className="w-base h-base mx-xsmall">
+      <Avatar user={user} font="inter-xsmall-semibold" />
+    </span>
+    {`${user?.first_name} ${user?.last_name}`}
```

Comment:
nit: This would properly look a bit weird in the case where first_name is undefined but last_name is defined